docusaurus/docs/dev-docs/api/plugins/admin-panel-api.md

```diff
@@ -96,43 +75,39 @@ export default {
 
 **Type**: `Function`
 
-Exposes the bootstrap function, executed after all the plugins are [registered](#register).
+This function is executed after all plugins are [registered](#register).
 
 Within the bootstrap function, a plugin can:
 
-* extend another plugin, using `getPlugin('plugin-name')`,
+* extend another plugin, using `getPlugin('plugin-id')`,
 * register hooks (see [Hooks API](#hooks-api))
 * [add links to a settings section](#settings-api)
 
 **Example:**
 
 ```js
-module.exports = () => {
-  return {
-    // ...
-    bootstrap(app) {
-      // execute some bootstrap code
-      app.injectContentManagerComponent('editView', 'right-links', { name: 'my-compo', Component: () => 'my-compo' })
-    },
-  };
+export default {
+  bootstrap(app) {
+    app.injectContentManagerComponent('editView', 'right-links', { name: 'my-component', Component: () => 'my-component' })
+  },
 };
 ```
 
-## Async function
-
-While [`register()`](#register) and [`bootstrap()`](#bootstrap) are lifecycle functions, `registerTrads()` is an async function.
+## Adding translations
```

Comment:
Using this exact title would break the flow of the table of content for this page:
![Screenshot 2023-08-21 at 14 58 48](https://github.com/strapi/documentation/assets/4233866/800c21ac-e2c9-4884-950b-ca28100e0161)
- We need either action verbs or nouns everywhere (technical writing best practice).
- The current table of content was designed around the nature of functions or files (entry file, lifecycle functions, async function(s)).
  - Is the `registerTrads()` not an async function anymore? (I'd say it still is, looking at the code 🤔)
  - Would there be another wording we could use to describe the nature of the function, in line with the other headings?
- Additionally, the section currently includes only one function, but this might change eventually, so we should try to keep future-proof titles (so we don't break header links too often).

Could you please think of another title to consider this context while preserving what you had in mind?  😊 If not, how would you restructure the whole page?

Reply:
I thought about it a lot already and I really couldn't come up with a category for it. "Async function" doesn't make sense to as a developer, because it describes what a function returns rather than what it does (add translations).

I think we could argue `registerTrads` is a lifecycle method to make this more concise, because it is called at some defined point?

At least for v5 there aren't any plans to extend the public API for now, so I don't think this is going to change anytime soon :)

Reply:
Great news that we don't need to think about extending this :D And thanks for the feedback and additional explanations :) I agree that we need to rethink the structure (and break links :D) here, then. 

Right now, I can think of 2 possible ways to update the structure:
- Solution A:
   - Entry file
   - Available functions (or maybe "available lifecycle functions"? 🤔)
      - register()
      - bootstrap()
      - registerTrads()
  - Available actions […]

- Solution B:
   - Entry file
   - Lifecycle functions
      - register()
      - bootstrap()
   - Specific functions
      - registerTrads() (or 'Adding translations with registerTrads()')
   - Available actions […]

What do you think about these?
 (We'll probably rethink this page while working on improving Plugins documentation next quarter and/or for v5 anyway ^^)